case-studies/antipatterns/classic/app/pods/side-effects/controller.js

```diff
@@ -1,23 +1,26 @@
-import Controller from '@ember/controller';
-import { computed } from '@ember/object';
+import Controller from "@ember/controller";
+import { computed } from "@ember/object";
 
 export default Controller.extend({
-  firstName: 'John',
-  lastName: 'Doe',
+  firstName: "John",
+  lastName: "Doe",
 
   fullNameLength: null,
```

Comment:
Great job putting the `fullNameLength` CP below the `fullName` CP, so that the order of code naturally follows the order of dependencies.

But you've left the original `fullNameLength: null` property behind. It will be overwritten by the CP, so it's OK, but you should remove it. ESLint can be configured to point out such issues.